projects/15_basic_material/main.cpp

```diff
@@ -480,6 +480,14 @@ void ProjApp::Setup()
         }
     }
 
+    const auto& cl_options = GetExtraOptions();
+    mMaterialIndex         = cl_options.GetExtraOptionValueOrDefault<int32_t>("material-index", 0);
```

Comment:
You can use `uint32_t` since the index variables are unsigned

Reply:
Ah yes, thanks!